fixtures/coverall/tests/bindings/test_coverall.kts

```diff
@@ -0,0 +1,31 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
```

Comment:
(note that these tests are from the commit in https://github.com/mozilla/uniffi-rs/pull/451, I just needed to get that fix in order to write the new test for this PR)